js/battle.js

```diff
@@ -4027,12 +4035,14 @@ var Battle = (function () {
 				var item = Tools.getItem(args[2]);
 				var effect = Tools.getEffect(kwargs.from);
 				var ofpoke = this.getPokemon(kwargs.of);
-				poke.item = '';
+				poke.item = item.name;
```

Comment:
That depends on your definition of accurate; prior to this change the tooltip couldn't distinguish between an unknown item or a removed item, but I agree that using the same property might be confusing and I'll think of another way to implement this.

Out of interest, in what way would the history from prior to the transfer of an item be useful?